scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/DefaultOpEnvironment.java

```diff
@@ -348,10 +378,10 @@ public Set<String> descriptions(String name) {
 	 * @return a {@link Set} of {@link String}s, one describing each
 	 *         {@link OpInfo} in {@code infos}.
 	 */
-	private Set<String> descriptions(Iterable<OpInfo> infos) {
+	private List<String> descriptions(Iterable<OpInfo> infos) {
 		return StreamSupport.stream(infos.spliterator(), true) //
 				.map(OpInfo::toString) //
-				.collect(Collectors.toSet());
+				.collect(Collectors.toList());
```

Comment:
I saw that you use `Collectors.toUnmodifiableList()` below... would it make sense to do so also here?